check_project.sh

```diff
@@ -151,6 +151,10 @@ check()
       MODULES_TO_CHECK_ARR+=("$SOURCE")
     fi
 
+    if [[ "$SOURCE" =~ helpers_emba_load_strict_settings ]]; then
+      # this file resets the IFS which is complained by semgrep - we exclude it from semgrep checks
```

Comment:
can't you use `// nosemgrep` here? [How to exclude semgrep rules for explicit lines](https://semgrep.dev/docs/ignoring-files-folders-code/)

Reply:
I have looked for this feature but have not found it :) Thanks for pointing me to it